src/components/sidebar/sideBar.scss

```diff
@@ -1,9 +1,8 @@
 .sidebar.MuiContainer-root {
   display: flex;
   flex-direction: column;
-  
+  padding: 1%;
```

Comment:
Let's use `px` or something else, instead of 1%
Currently, it doesn't show 1% of what